### DISCUSSION TITLE
1.3.0

### DISCUSSION THREAD

**Discussion by buchdag** [nginx-proxy/nginx-proxy#2221] [Announcements]
[![nginx 1.23.4](https://img.shields.io/badge/nginx-1.23.4-brightgreen.svg)](https://nginx.org/en/CHANGES)
## New features
* feat: Add support for HTTP keep-alive between the proxy and upstream by @rhansen in https://github.com/nginx-proxy/nginx-proxy/pull/1934
* feat: Add support for HTTP load balancing between the proxy and upstream server groups by @SchoNie in https://github.com/nginx-proxy/nginx-proxy/pull/2173
## Fixes
* fix: Improve handling of unknown hosts and missing certs by @rhansen in https://github.com/nginx-proxy/nginx-proxy/pull/2186
* fix: Remove `default_server` listen option from fallback server by @rhansen in https://github.com/nginx-proxy/nginx-proxy/pull/2214
## Other changes
* build: bump nginx from 1.23.3-alpine to 1.23.4-alpine by @dependabot in https://github.com/nginx-proxy/nginx-proxy/pull/2206
* build: docker-gen 0.10.2 -> 0.10.3 by @buchdag in https://github.com/nginx-proxy/nginx-proxy/pull/2207
* build: dockergen 0.10.3 -> 0.10.4 by @buchdag in https://github.com/nginx-proxy/nginx-proxy/pull/2220
* build: bump golang from 1.20.2-alpine to 1.20.3-alpine by @dependabot in https://github.com/nginx-proxy/nginx-proxy/pull/2213
* ci: bump pytest from 7.2.2 to 7.3.0 in /test/requirements by @dependabot in https://github.com/nginx-proxy/nginx-proxy/pull/2216
* ci: bump pytest from 7.3.0 to 7.3.1 in /test/requirements by @dependabot in https://github.com/nginx-proxy/nginx-proxy/pull/2219

## New Contributors
* @SchoNie made their first contribution in https://github.com/nginx-proxy/nginx-proxy/pull/2173

**Full Changelog**: https://github.com/nginx-proxy/nginx-proxy/compare/1.2.3...1.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nginx-proxy/nginx-proxy/releases/tag/1.3.0'>1.3.0</a>.</em>

**Comment by SchoNie** [comment]
${NGINX_PROXY_VERSION} reports as: 1.2.3-24-gb4efff0 in this 1.3.0 image.
`ENV NGINX_PROXY_VERSION=1.2.3-24-gb4efff0 DOCKER_GEN_VERSION=0.10.4 DOCKER_HOST=unix:///tmp/docker.sock`

I can confirm nginx is 1.23.4 and dockergen is 0.10.4 so seems just cosmetic from the git-describe?

Seems to be only in the [latest alpine](https://hub.docker.com/layers/nginxproxy/nginx-proxy/alpine/images/sha256-85e5117f4c120ab0bd4bf1c571fd0a1269168911e1b825a209ef9ab4e5372e5c?context=explore)

**Reply by buchdag** [reply]
I think the build for the Alpine version triggered by the push to main just before the release somehow completed after the one triggered by the 1.3.0 release and overwrote the `alpine` tag.

I manually re-triggered a build, everything should be fine on `latest` / `alpine` now.
